templates/web/fixmystreet.com/static/unresponsive.html

```diff
@@ -0,0 +1,37 @@
+[% INCLUDE header.html
+    title = 'Unresponsive councils: What you can do', bodyclass = 'unresponsive-council fullwidthpage'
+%]
+
+<div class="unresponsive-council__header">
+  <h1>Bad news :(</h1>
+  <p><strong class="unresponsive-council-name">[% body.name %]</strong> no
+  longer accepts reports from FixMyStreet.</p>
```

Comment:
I think this is potentially misleading - the council might only refuse reports in certain categories but the wording makes it seem like they refuse *all* reports.